src/domain/Request.ts

```diff
@@ -0,0 +1,8 @@
+import { Model } from 'objectmodel';
+
+export class Request extends Model({
```

Comment:
This model is incorrect. You have to send an event when a rental is requested and a rental is accepted.

Reply:
And declined!